doitlive.py

```diff
@@ -22,9 +22,13 @@
 from code import InteractiveConsole
 from tempfile import NamedTemporaryFile
 from collections import OrderedDict
+from termutils import raw_mode
 
 import click
-from click import echo, style, secho, getchar
+if int(click.__version__[0]) >= 5:
+    click.disable_unicode_literals_warning = True  # this fixes #12
```

Comment:
I'm still not sure this is the best fix; the click docs recommend using the `u` prefix explicitly to avoid subtle bugs, so we should probably follow that recommendation.